### DISCUSSION TITLE
Internal server error

### DISCUSSION THREAD

**Discussion by staocube88** [apache/couchdb#4816] [Q&A]
Couchdb deployment with persistence successful but its internal server error when i try to do anything

root@rm-couchdb-couchdb-0:/# curl -s \
    http://127.0.0.1:5984/_cluster_setup \
    -X POST \
    -H "Content-Type: application/json" \
    -d '{"action": "finish_cluster"}' \
    -u admin
Enter host password for user 'admin':
{"ok":true}
root@rm-couchdb-couchdb-0:/# curl -X PUT http://admin:Gh2KYQZHVZthWVFLuhbx@localhost:5984/users
{"error":"error","reason":"internal_server_error"}

**Comment by big-r81** [comment]
Hi,

look into the log file, there should be additional informations.
